data/json/mutations.json

```diff
@@ -3926,10 +3926,7 @@
         "cancels" : ["VEGETARIAN", "HERBIVORE", "RUMINANT",  "GRAZER"],
         "leads_to" : ["SAPROVORE"],
         "category" : ["MUTCAT_LIZARD", "MUTCAT_BEAST", "MUTCAT_SPIDER", "MUTCAT_CHIMERA", "MUTCAT_RAPTOR", "MUTCAT_FELINE"],
-        "vitamin_rates": [
-            [ "VitA", -999 ],
-            [ "VitC", -999 ]
-        ]
+        "vitamin_rates": [ [ "vitC", -2 ] ]
```

Comment:
vitamin rates are expressed in minutes not hours with a default rate of 1 unit every 60 minutes.  	Fixed in f40993b